autora_closed_loop_demo/researcher_hub/autora_workflow.py

```diff
@@ -160,42 +167,39 @@ def runner_on_state(conditions):
     for item in data:
         result.append(json.loads(item))
     return Delta(experiment_data=pd.DataFrame(result))
-'''
+"""
+
 
 def trial_list_to_experiment_data(trial_sequence):
     """
     Parse a trial sequence (from jsPsych) into dependent and independent variables
-    independent: S1, S2
-    dependent: rt
+    independent: coherence_ratio, motion_direction
+    dependent: d_prime
     """
-    res_dict = {
-        'S1': [],
-        'S2': [],
-        'rt': []
-    }
+    res_dict = {"coherence_ratio": [], "motion_direction": [], "d_prime": []}
     for trial in trial_sequence:
         # Filter trials that are not ROK (instructions, fixation, ...)
-        if trial['trial_type'] != 'rok':
+        if trial["trial_type"] != "rok":
             continue
         # Filter trials without rt
-        if 'rt' not in trial or trial['rt'] is None:
+        if "d_prime" not in trial or trial["d_prime"] is None:
             continue
         # the intensity is equivalent to the number of oobs (set in sweetBean script)
         # rt is a default value of every trial
-        s1 = trial['number_of_oobs'][0] # first value in list
-        s2 = trial['number_of_oobs'][1]
-        rt = trial['rt']
-        #key = trial['key_press'] oder trial['correct']
-        
-        res_dict['S1'].append(int(s1))
-        res_dict['S2'].append(int(s2))
-        res_dict['rt'].append(float(rt))
-    
+        s1 = trial["number_of_oobs"][0]  # first value in list
+        s2 = trial["number_of_oobs"][1]
```

Comment:
name variable "motion_direction" instead of s2 here and in line 195 for consistency

Reply:
Done. But not sure if this still complete.